README.md

```diff
@@ -194,6 +194,22 @@ Once the database schema has been created, you can open the _Federation_ tab fro
 
 The module lets you create, read, update and delete all the RP you want. To see the client id and the client secret press the show button.
 
+### Client self registration
+
+You can allow users to register their own clients.
```

Comment:
Maybe also add description on how to gracefully disable this feature. Is it best to leave out 'permissions' config or leave ['permissions']['client'] empty?

Reply:
Make sense